lib/subsocial_sdk.dart

```diff
@@ -261,6 +297,38 @@ class Subsocial {
     return val;
   }
 
+  Future<PostReactionUpdated> updatePostReaction({
+    required PostId postId,
+    required ReactionId reactionId,
+    required Reaction_Kind kind,
```

Comment:
Why is it `Reaction_Kind` not `ReactionKind`

Reply:
it is how the codegen generated it, I probably would create a type alias for it.